variables.tf

```diff
@@ -8,12 +8,28 @@ variable "enforce_domain_name" {
 
 variable "enforce_group_id" {
   type        = string
-  description = "Enforce IAM group ID to bind your AWS account to"
+  description = "DEPRECATED: Please use 'enforce_group_ids'. Enforce IAM group ID to bind your AWS account to"
   sensitive   = false
   nullable    = false
 
   validation {
     condition     = length(regexall("^[a-f0-9]{40}(\\/[a-f0-9]{16})*$", var.enforce_group_id)) == 1
-    error_message = "enforce_group_id must be a valid group id"
+    error_message = "The value 'enforce_group_id' must be a valid group id."
+  }
+}
+
+variable "enforce_group_ids" {
+  type        = list(string)
+  description = "Enforce IAM group IDs to bind your AWS account to. If both 'enforce_group_id' and 'enforce_group_ids' are specified, 'enforce_group_id' is ignored."
+  sensitive   = false
+  default     = []
+
+  validation {
+    condition     = length(var.enforce_group_ids) > 0
```

Comment:
Can this be `length(var.enforce_group_ids) > 0 || var.enforce_group_id != ""`. If the deprecated param is set, and not the new one, this condition cause breakage I think

Reply:
sadly it cant.
terraform validation doesnt allow cross variable reference.